### PR TITLE
consolidate all deprecated flags under one header in release notes

### PR DESCRIPTION
**Pull request by harshit-gangal**
## Description

This PR reorders the deprecated flags from different sections under one header.
This could help readers to know all the deprecated flags under one section.

## Checklist

-   [ ] "Backport to:" labels have been added if this change should be back-ported
-   [ ] Tests were added or are not required
-   [ ] Did the new or modified tests pass consistently locally and on the CI
-   [ ] Documentation was added or is not required

### CODE REVIEW

File: changelog/17.0/17.0.0/summary.md [hunk 1]
```diff
@@ -304,6 +301,7 @@ Prior to v17, this data was not available via the Prometheus backend. In v17, wo
 vttablet_v_replication_stream_state{counts="1",state="Running",workflow="commerce2customer"} 1
 ```
 
+<<<<<<< Updated upstream
 ## <a id="deprecations-and-deletions"/> Deprecations and Deletions
```
Comment (frouioui) [inline changelog/17.0/17.0.0/summary.md:305]: we should remove this merge conflict

Comment (harshit-gangal) [reply changelog/17.0/17.0.0/summary.md:305]: removed

Comment (harshit-gangal) [reply changelog/17.0/17.0.0/summary.md]: :( how did it get there

**Review by frouioui — APPROVED** [review]
LGTM

**Review by mattlord — APPROVED** [review]

**Pull request merged**